pom.xml

```diff
@@ -621,6 +621,9 @@
         </profile>
         <profile>
             <id>kafka-0.10-dev</id>
+            <activation>
```

Comment:
This is probably better to be done in a separate PR, either before and after your PR.  I would prefer to track the version upgrade as its own PR with its own version/tag.